track_info.c

```diff
@@ -46,8 +46,12 @@ static void track_info_free(struct track_info *ti)
 
 struct track_info *track_info_new(const char *filename)
 {
+	static uint64_t cur_uid = 0;
+	cur_uid++;
```

Comment:
cmus does not implement the tracklist feature at the moment. If it ever gets implemented one might have to revisit this point.